src/open-data/Services.csv

```diff
@@ -1,314 +1,314 @@
+Microsoft.AzureArcData;microsoft.azurearcdata/sqlmanagedinstances;Azure Arc;Multicloud;Microsoft;Cloud Provider;Both;SaaS
+Microsoft.AzureArcData;microsoft.azurearcdata/sqlserverinstances;Azure Arc;Multicloud;Microsoft;Cloud Provider;On-premises;SaaS
+Microsoft.AzureStackHCI;microsoft.azurestackhci/clusters;Azure Stack HCI;Multicloud;Microsoft;Cloud Provider;On-premises;IaaS
+Microsoft.BareMetalInfrastructure;microsoft.baremetalinfrastructure/baremetalstorageinstances;BareMetal Infrastructure;Compute;Microsoft;Cloud Provider;Both;IaaS
+Microsoft.Batch;microsoft.batch/batchaccounts;Batch;Compute;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.Bing;microsoft.bing/accounts;Microsoft Bing;Web;Microsoft;Cloud Provider;Cloud;SaaS
+Microsoft.BotService;microsoft.botservice/botservices;Azure AI Bot Service;AI and Machine Learning;Microsoft;Cloud Provider;Cloud;SaaS
+Microsoft.Cache;microsoft.cache/redis;Azure Cache for Redis;Storage;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.EventGrid;microsoft.cache/redis;Azure Cache for Redis;Storage;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.Cache;microsoft.cache/redisenterprise;Azure Cache for Redis;Storage;Microsoft;Cloud Provider;Cloud;PaaS
+microsoft.cdn;microsoft.cdn/cdnwebapplicationfirewallpolicies;Content Delivery Network;Networking;Microsoft;Cloud Provider;Cloud;SaaS
+Microsoft.Cdn;microsoft.cdn/profiles;Content Delivery Network;Networking;Microsoft;Cloud Provider;Cloud;SaaS
+microsoft.certificateregistration;microsoft.certificateregistration/certificateorders;Azure App Service;Web;Microsoft;Cloud Provider;Both;PaaS
+microsoft.classiccompute;microsoft.classiccompute/domainnames;Virtual Machines;Compute;Microsoft;Cloud Provider;Both;IaaS
+microsoft.classiccompute;microsoft.classiccompute/virtualmachines;Virtual Machines;Compute;Microsoft;Cloud Provider;Both;IaaS
+Microsoft.ClassicNetwork;microsoft.classicnetwork/ipv4vip;Virtual Network;Networking;Microsoft;Cloud Provider;Both;IaaS
+Microsoft.ClassicNetwork;microsoft.classicnetwork/publicip;Virtual Network;Networking;Microsoft;Cloud Provider;Both;IaaS
+Microsoft.ClassicNetwork;microsoft.classicnetwork/publicipaddresses;Virtual Network;Networking;Microsoft;Cloud Provider;Cloud;IaaS
+Microsoft.ClassicStorage;microsoft.classicstorage/storageaccounts;Storage Accounts;Storage;Microsoft;Cloud Provider;Cloud;IaaS
+Microsoft.CognitiveServices;microsoft.cognitiveservices/accounts;Azure AI Services;AI and Machine Learning;Microsoft;Cloud Provider;Cloud;SaaS
+microsoft.communication;microsoft.communication/communicationservices;Azure Communication Services;Web;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.EventGrid;microsoft.communication/communicationservices;Azure Communication Services;Web;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.Compute;microsoft.compute/capacityreservationgroups;Virtual Machines;Compute;Microsoft;Cloud Provider;Cloud;PaaS

```

Comment:
These are for reserving IaaS capacity, so probably IaaS?

Reply:
Corrected to IaaS